main.tf

```diff
@@ -308,3 +308,49 @@ resource "aws_codepipeline" "default" {
     }
   }
 }
+
+resource "random_string" "webhook_secret" {
+  count  = local.enabled && var.webhook_enabled ? 1 : 0
+  length = 32
+
+  # Special characters are not allowed in webhook secret (AWS silently ignores webhook callbacks)
+  special = false
+}
+
+locals {
```

Comment:
move this local vars to all other `locals` https://github.com/cloudposse/terraform-aws-cicd/blob/master/main.tf#L7